com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -1150,12 +1144,11 @@ static void CheckAndApplyDebugSettings(ref RenderingData renderingData)
                 if (!debugDisplaySettings.IsPostProcessingAllowed)
                     cameraData.postProcessEnabled = false;
 
-                cameraData.cameraTargetDescriptor = CreateRenderTextureDescriptor(cameraData.camera,
-                    cameraData.renderScale,
-                    cameraData.isHdrEnabled,
-                    msaaSamples,
-                    true,
-                    cameraData.requiresOpaqueTexture);
+                GraphicsFormat hdrFmt = RenderingUtils.SupportsGraphicsFormat(GraphicsFormat.R16G16B16A16_SFloat, FormatUsage.Linear | FormatUsage.Render) ?
+                    GraphicsFormat.R16G16B16A16_SFloat
+                    : SystemInfo.GetGraphicsFormat(DefaultFormat.HDR);
+                cameraData.cameraTargetDescriptor.graphicsFormat = cameraData.isHdrEnabled ? hdrFmt : SystemInfo.GetGraphicsFormat(DefaultFormat.LDR);
+                cameraData.cameraTargetDescriptor.msaaSamples = msaaSamples;
```

Comment:
Why not use same logic as in https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs#L486

Reply:
I don't want to create a new descriptor, I just want to override a few key values on the descriptor. The original code which creates a new descriptor breaks XR because in cameraTargetDescriptor has been set up to use a texture array etc. 

However I think your point is valid, and I shouldn't duplicate this logic - I should probably refactor `CreateRenderTextureDescriptor` so that there are specific utility functions that are responsible for determining `graphicsFormat`.

Reply:
Fixed in this commit https://github.com/Unity-Technologies/Graphics/pull/4089/commits/2a5e271e9bb38bc6bae66ffc33cd451d029ef824